File: light-bulb.m
total_expenses = 61;
room1_lamps = 10;
room1_days = 21;
room1_hours = 12;
room2_lamps = 20;
room2_days = 30;
room2_hours = 8;

total_lamps = room1_lamps + room2_lamps;
total_days = room1_days + room2_days;
total_hours = room1_hours + room2_hours;

room1_expenses = (room1_lamps/total_lamps) * (room1_days/total_days) * (room1_hours/total_hours) * total_expenses;
room2_expenses = (room2_lamps/total_lamps) * (room2_days/total_days) * (room2_hours/total_hours) * total_expenses;

fprintf('Expenses for Room 1: %.2f rubles\n', room1_expenses);
fprintf('Expenses for Room 2: %.2f rubles\n', room2_expenses);

File: light-bulb.cpp
#include <iostream>
using namespace std;

int main() {
    double totalExpenses = 61.0;
    
    int lampsRoom1 = 10;
    int daysRoom1 = 21;
    int hoursRoom1 = 12;
    
    int lampsRoom2 = 20;
    int daysRoom2 = 30;
    int hoursRoom2 = 8;
    
    double totalLamps = lampsRoom1 + lampsRoom2;
    double totalDays = daysRoom1 + daysRoom2;
    double totalHours = hoursRoom1 + hoursRoom2;
    
    double expensesRoom1 = (lampsRoom1 / totalLamps) * (daysRoom1 / totalDays) * (hoursRoom1 / totalHours) * totalExpenses;
    double expensesRoom2 = (lampsRoom2 / totalLamps) * (daysRoom2 / totalDays) * (hoursRoom2 / totalHours) * totalExpenses;
    
    cout << "Расходы на освещение первого помещения: " << expensesRoom1 << " рублей" << endl;
    cout << "Расходы на освещение второго помещения: " << expensesRoom2 << " рублей" << endl;
    
    return 0;
}
